pages/common/ansible.md

```diff
@@ -0,0 +1,27 @@
+# ansible
+
+> Manage groups of computers remotely over SSH
+
+- Display a list of hosts belonging to a group:
+
+`ansible {{group}} --list-hosts`
+
+- Ping a group of hosts:
+
+`ansible {{group}} -m ping`
+
+- Display facts about a group of hosts:
+
+`ansible {{group}} -m setup`
+
+- Execute a command (using the command module) on a group of hosts:
```

Comment:
The following considerations were made for this description:

- The intent ("execute a command") should have higher priority over the mechanics ("command module"), as the user will be scanning the docs with a specific intention in mind. Thus, "Execute a command" is placed at the beginning of the sentence.

- The description should provide a hint at the correct mental model (as per comments by @waldyrious)